src/modules/postComments.js

```diff
@@ -0,0 +1,19 @@
+import { involvementURL, involvementApiKey } from './apiCollection.js';
+
+const commentURL = `${involvementURL}${involvementApiKey}/comments`;
+
+const postComments = async (ID, name, comment) => {
+  await fetch(commentURL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: ID,
+      username: name,
+      comment,
+    }),
+  });
+};
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the comments :clap: However it would be nice to make sure that the `name` and the `comment` are not empty before send comment. That will avoid having empty comments in the comments session.